dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -507,6 +511,13 @@ internal class DatadogRumMonitor(
         }
     }
 
+    internal fun sendSdkInitEvent() {
+        val eventTime = getEventTime(emptyMap())
```

Comment:
I think we don't need this line. `getEventTime(emptyMap())` will return `Time()` object, but the same object will be created at the `SdkInit` constructor with default argument.